src/northbridge/amd/amdfam10/northbridge.c

```diff
@@ -536,6 +536,8 @@ static void amdfam10_create_vga_resource(struct device *dev, unsigned int nodeid
 
 	printk(BIOS_DEBUG, "VGA: %s (aka node %d) link %d has VGA device\n", dev_path(dev), nodeid, link->link_num);
 	set_vga_enable_reg(nodeid, link->link_num);
+
+	reserved_ram_resource(dev, 6, 0xa0000 >> 10, 0x20000 >> 10);
```

Comment:
I just removed it in the other PR :) So what and why exactly do we do here?

Reply:
This is added for case with dGPU and disabled Aspeed - in that case this region wouldn't be reserved.